lib/lightning/version_control/github_token.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Lightning.VersionControl.GithubToken do
+  @moduledoc """
+  A module that `uses` Joken to handle building and signing application 
+  tokens for communicating with github
+  """
+  use Joken.Config
+end
```

Comment:
Just a suggestion, if this module is only used inside the client (and we're not writing tests directly against it), it should live nested inside `GithubClient`.